src/Solnet.Rpc/Utilities/AddressExtensions.cs

```diff
@@ -19,6 +20,40 @@ public static class AddressExtensions
         /// The bytes of the `ProgramDerivedAddress` string.
         /// </summary>
         private static readonly byte[] ProgramDerivedAddressBytes = Encoding.UTF8.GetBytes("ProgramDerivedAddress");
+        private static byte[] EncodeRustString(string data)

```

Comment:
Can't the helper method in the utilities be used so we don't have this kind of code duplication?

Reply:
The same can be said for the functions below

Reply:
Eliminated that and the write functions below but kept createwithseed bc that doesn't exist elsewhere and that class is where other things from the pubkey.rs are implemented

Reply:
moved createwithseed to the utilities helper class to avoid circular dependency